src/xcode/ENA/ENA/Source/Services/Risk/Calculation/RiskCalculationExposureWindow.swift

```diff
@@ -49,11 +49,11 @@ final class RiskCalculationExposureWindow: Codable {
 		self.init(exposureWindow: exposureWindow, configuration: configuration)
 	}
 
+	// encode data to json - used for debugging help
 	func encode(to encoder: Encoder) throws {
 		var container = encoder.container(keyedBy: CodingKeys.self)
 
 		try container.encode(exposureWindow, forKey: .exposureWindow)
-		try container.encode(configuration, forKey: .configuration)
```

Comment:
I tried this before. If we don't encode the configuration, the decoding fails because of the missing configuration and we're not able to get the most recent risk calculation from the store. It always returns `nil`. I couldn't come up with a quick solution last time, that's why the configuration is still encoded with every window.